src/Apps/Collect/CollectApp.tsx

```diff
@@ -63,6 +63,8 @@ export const CollectAppFragmentContainer = createFragmentContainer(
         }
         sort: { type: "String", defaultValue: "-partner_updated_at" }
         price_range: { type: "String" }
+        artist_id: { type: "String" }
```

Comment:
Since I'm reusing the collect app's filter components, I added these two filters to the query so that we can offer a filter on these.

Reply:
At some point it would be good if we abstracted this filter away somehow. Between this and Artist app there's now a lot of shared code (but don't really know the data particulars enough to know if this is even possible...)

Reply:
Which brings to mind recent work like the zerostate which ideally would live in a centralized place: 

<img width="1257" alt="screen shot 2018-10-12 at 4 02 52 pm" src="https://user-images.githubusercontent.com/236943/46897595-73bc2280-ce38-11e8-831c-6082e5b8e73f.png">

Reply:
I think the collect app should be the canonical place for this, as it will include _all_ our possible filters.

Then, clients could possibly pass in a 'stuck' param (so on the artist page it would be `artist_id`, on a gene page `gene_id`). Clients may also want to be able to pass in zero state messaging specific for their use.

Reply:
or rather, we should move the Collect app/filter into its own component (possibly some other abstractions/refactors needed there), and then we would attempt to use that in our different contexts.

Reply:
Yah I feel like we know enough about its behavior at this point to try

Reply:
Definitely. I would prefer to make that its own work item/refactor however. This PR is already pretty big (+ i have more local changes), and this is just something for internal use. The filter component for the artist page itself can be rewritten/simplified (that was just our first working version after struggling!), and then it was mostly copied over to Collect. So I think it's a worth a separate item to revisit the implementation and try to consolidate _and_ improve.

Reply:
Hey @matt, I want to point out I've been working on a refactor of Collect that extracts filter UI components to make them reusable independently from relay containers.
This work is part building Collection pages which are needed to launch in time for Buy Now, so it's pretty time sensitive.

Let's sync later today to align our efforts.
Here's the PR: https://github.com/artsy/reaction/pull/1408.